rc/filetype/apl.kak

```diff
@@ -26,6 +27,10 @@ hook -group apl-highlight global WinSetOption filetype=apl %{
     hook -once -always window WinSetOption filetype=.* %{ remove-highlighter window/apl }
 }
 
+@
```

Comment:
(mega nit, feel free to ignore) I think in this case we can use `%< >` to use the `m` command